### DISCUSSION TITLE
xrdp v0.9.21.1

### DISCUSSION THREAD

**Discussion by metalefty** [neutrinolabs/xrdp#2471] [Announcements]
# Release notes for xrdp v0.9.21.1 (2022/12/13)

This release only includes following fix for packagers. Packagers try to build xrdp on distributions _other than_ Arch Linux, Debian, SUSE, Red Hat(ish), FreeBSD and macOS may be required to use this release.

## Changes for packagers or developers
* Add missing xrdp-sesman.system to distributed tarball (#2466 #2468)

<hr /><em>This discussion was created from the release <a href='https://github.com/neutrinolabs/xrdp/releases/tag/v0.9.21.1'>xrdp v0.9.21.1</a>.</em>

**Comment by Benzitczo** [comment]
problem no working when:

1) open desktop remote
2) linux xdrp
3) close desktop remote problem.

Please me message private discord
